api/src/main/resources/workbench-api.yaml

```diff
@@ -0,0 +1,6369 @@
+          description: User doesn't have the ACCESS_CONTROL_ADMIN authority
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+        400:
+          description: No module exists with name submitted
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/users/{userId}":
+    get:
+      tags:
+      - profile
+      description: 'Returns a user''s profile for review.  Requires ACCESS_CONTROL_ADMIN
+        authority.
+
+'
+      parameters:
+      - "$ref": "#/parameters/userId"
+      operationId: getUser
+      responses:
+        200:
+          description: A user's profile
+          schema:
+            "$ref": "#/definitions/Profile"
+        403:
+          description: User doesn't have the ACCESS_CONTROL_ADMIN authority
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/users/list":
+    get:
+      tags:
+      - profile
+      description: 'Returns a list of profiles for users to be reviewed. Requires
+        ACCESS_CONTROL_ADMIN authority.
+
+'
+      operationId: getAllUsers
+      responses:
+        200:
+          description: A list of users
+          schema:
+            "$ref": "#/definitions/UserListResponse"
+        403:
+          description: User doesn't have the ACCESS_CONTROL_ADMIN authority
+          schema:
+            "$ref": "#/definitions/ErrorResponse"
+  "/v1/admin/workspaces/{workspaceNamespace}/{workspaceId}/review":
+    parameters:
+    - "$ref": "#/parameters/workspaceNamespace"
+    - "$ref": "#/parameters/workspaceId"
+    post:
+      tags:
+      - workspaces
+      consumes:
+      - application/json
+      description: Sets a research purpose review result.
+      operationId: reviewWorkspace
+      parameters:
+      - in: body
+        name: review
+        description: result of the research purpose review
+        schema:
+          "$ref": "#/definitions/ResearchPurposeReviewRequest"
+      responses:
+        200:
+          description: success
+          schema:
+            "$ref": "#/definitions/EmptyResponse"
+  "/v1/admin/clusters/updateConfig":
+    post:
+      summary: Sets default cluster creation request parameters for a user.
+      description: 'Clusters are created with a default machine configuration. Setting
+        this override changes that configuration for subsequent cluster creations.
+        This change only takes effect after a new cluster creation, e.g. due to standard
+        cluster expiration (~2w) or via manual reset. Requires DEVELOPER authority.
+
+'
+      operationId: updateClusterConfig
+      tags:
+      - cluster
+      parameters:
+      - in: body
+        name: body
+        description: Cluster config update request.
+        schema:
+          "$ref": "#/definitions/UpdateClusterConfigRequest"
+      responses:
+        200:
+          description: success
+          schema:
+            "$ref": "#/definitions/EmptyResponse"
+  "/v1/cron/auditBigQuery":
```

Comment:
An important comment was dropped here:

```
  # Note: all requests tagged as "cron" must have the header X-Appengine-Cron:
  # true, which app engine itself only sets when invoking as a cronjob.
  # See https://cloud.google.com/appengine/docs/standard/java/config/cron#securing_urls_for_cron
  # and o.p.w.interceptors.CronInterceptor which implements the header check.
```

Please preserve this

Reply:
How about making this a [tag description](https://swagger.io/docs/specification/grouping-operations-with-tags/)? Otherwise it'll get lost again when we translate to OpenAPI 3. There was a brief [conversation](https://pmi-engteam.slack.com/archives/CHRN2R51N/p1576941635031300) about this, and we plan to re-add all the comments at the end.

Reply:
This one would be reasonable as a description tag